src/CommandHandler/Wishlist/AddSelectedProductsToCartHandler.php

```diff
@@ -90,5 +78,10 @@ private function addProductToWishlist(WishlistItem $wishlistProduct): void
 
         $this->orderModifier->addToOrder($cart, $cartItem);
         $this->orderRepository->add($cart);
+
+        if (false === $this->flashBag->has('success')){
+            $this->flashBag->add('success', $this->translator->trans('bitbag_sylius_wishlist_plugin.ui.added_to_cart'));
```

Comment:
We should throw an exception here and move flashbag to a controller.

Reply:
Done.